vcsclient/gitlab.go

```diff
@@ -23,6 +23,13 @@ type GitLabClient struct {
 	logger   Log
 }
 
+func extractTimeWithFallback(timeObject *time.Time) time.Time {
+	if timeObject == nil {
+		return time.Time{}
+	}
+	return timeObject.UTC()
+}
```

Comment:
Let's put this in a common location (like vcsclient.go) and use it in azurerepos.go too